src/client/components/pages/entity-revisions.js

```diff
@@ -118,10 +118,15 @@ class EntityRevisions extends React.Component {
 		return (
 			<div>
 				{this.renderHeader()}
-				<h2>Revision History</h2>
-				<ListGroup>
-					{revisions.map(this.renderRevision)}
-				</ListGroup>
+				{revisions.length > 0 ?
+					<div>
+						<h2>Revision History</h2>
+						<ListGroup>
+							{revisions.map(this.renderRevision)}
+						</ListGroup>
+					</div> :
+					<h2>Revision History not found.</h2>
```

Comment:
I think this would be better phrased as "No revision to show", meaning we did find what you are looking for (compared for example to typing a random number in the address bar), but there is nothing to show.